src/script/interpreter.cpp

```diff
@@ -969,6 +969,19 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                 }
                 break;
 
+				// memory store
```

Comment:
the indentation is way off

Reply:
I don't know why github shows it like this. In VSCode it seems ok

![image](https://user-images.githubusercontent.com/765487/116011873-db6d9e00-a62f-11eb-8651-2565e5d62f32.png)

Reply:
it seems the classical problem between tabs and spaces

try to use these setting on vscode:
- Editor: Detect Indentation
- Editor: Tab Size 4
- Editor: Insert Spaces

Reply:
I hate spaces :)